src/mono/wasm/runtime-test.js

```diff
@@ -49,28 +47,16 @@ if (is_browser) {
 	let consoleWebSocket = new WebSocket(consoleUrl);
 	consoleWebSocket.onopen = function(event) {
 		proxyJson(function (msg) { consoleWebSocket.send (msg); });
-		globalThis.testConsole.log("browser: Console websocket connected.");
+		console.log("browser: Console websocket connected.");
```

Comment:
Not opposed to this, but do you know whether there were specific use cases for testConsole that won't be addressed as a result of this change?

Reply:
I didn't see any uses for it when I did a search within the repo but I am not entirely sure. Could it be referenced indirectly somehow from C#? Although looks like the tests pass, so it probably isn't used anywhere.

Also this was a change that Larry did in his PR upon which this one is loosely based. @lewing do you know if this is used anywhere besides this file?